src/test/fuzz.cpp

```diff
@@ -157,6 +169,16 @@ fuzz(std::string const& filename, el::Level logLevel,
     {
         clock.crank(true);
     }
+
+    // properly shut everything down
+    app1->getWorkScheduler().shutdown();
+    app2->getWorkScheduler().shutdown();
+    while (
```

Comment:
this is ugly/weird, sharing the clock between applications is very error prone and I suspect leads to all sorts of random behaviors.
Switch to using the `Simulation` class

Reply:
Done